src/components/clickable-image/ClickableImage.tsx

```diff
@@ -0,0 +1,34 @@
+import { FC } from 'react'
+
+import { Box } from '@mui/material'
+
+import AppButton from '~/components/app-button/AppButton'
+
+import { ButtonVariantEnum } from '~/types'
+
+import { styles } from '~/components/clickable-image/ClickableImage.styles'
+
+interface ClickableImageProps {
+  clickFunction?(image: string): void
+  image: string
+}
+
+const ClickableImage: FC<ClickableImageProps> = ({
+  clickFunction,
+  image,
+  children
+}) => {
+  return (
+    <AppButton
+      data-testid='sidebar-image'
```

Comment:
I guess, you can get it with screen.getByText() method in testing

Reply:
> I guess, you can get it with screen.getByText() method in testing

I use it in the `AboutChatSidebar` component to count the number of images played. So I leave it as it is😅

Reply:
> No, I mean, that you added data-testid to get this element in the test, but you can also get it by using getByText() method with button text, so I think, this data-testid is redundant. Try to use getByText()

fixed